storybook/qmlTests/tests/tst_OwnerTokenWelcomeView.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import QtTest 1.15
+
+import StatusQ 0.1
+
+import SortFilterProxyModel 0.2
+import AppLayouts.Communities.panels 1.0
+import AppLayouts.Communities.views 1.0
+
+import Models 1.0
+
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+Item {
+
+    id: root
+    width: 600
+    height: 800
+
+    Component {
+        id: componentUnderTest
+
+        OwnerTokenWelcomeView {
+
+            anchors.fill: parent
+
+            communityLogo: ModelsData.icons.superRare
+            communityColor: "Light pink"
+            communityName: "SuperRare"
+
+            }
+        }
+
+    property OwnerTokenWelcomeView name: OwnerTokenWelcomeView
+
+    SignalSpy {
+        id: nextClickSpy
+        target: componentUnderTest
```

Comment:
here can you please use controlUnderTest instead of componentUnderTest, and move it within the Testcase? else all looks okay